# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1767286044 -3600
# Parent  b4455c4ab1e85b8023065f8380994b407d220f23
NNN30 - On message drag to compose window, prepend 'Re: ' if it's a reply.

diff --git a/mail/components/compose/content/MsgComposeCommands.js b/mail/components/compose/content/MsgComposeCommands.js
--- a/mail/components/compose/content/MsgComposeCommands.js
+++ b/mail/components/compose/content/MsgComposeCommands.js
@@ -10544,8 +10544,11 @@ var envelopeDragObserver = {
           isValidAttachment = true;
           const msgHdr =
             MailServices.messageServiceFromURI(data).messageURIToMsgHdr(data);
           prettyName = msgHdr.mime2DecodedSubject;
+          if (msgHdr.flags & Ci.nsMsgMessageFlags.HasRe) {
+            prettyName = "Re: " + prettyName;
+          }
           if (Services.prefs.getBoolPref("mail.forward_add_extension")) {
             prettyName += ".eml";
           }
 
